bagel/cli.py

```diff
@@ -98,14 +99,12 @@ def pheno(
     pheno_df = file_utils.load_tabular(pheno)
     pheno_utils.validate_inputs(data_dictionary, pheno_df)
 
-    # NOTE: `space` determines the amount of padding (in num. characters) before the file paths in the print statement.
-    # It is currently calculated as = (length of the longer string, including the 3 leading spaces) + (2 extra spaces)
-    space = 25
-    print(
-        "Processing phenotypic annotations:\n"
-        f"   {'Tabular file (.tsv):' : <{space}} {pheno}\n"
-        f"   {'Data dictionary (.json):' : <{space}} {dictionary}\n"
-    )
+    # NOTE: `width` determines the amount of padding (in num. characters) before the file paths in the print statement.
+    # It is calculated as = length of the longer string + 2 extra spaces
+    width = 26
+    logger.info("Processing phenotypic annotations:")
+    logger.info("%-*s%s", width, "Tabular file (.tsv):", pheno)
```

Comment:
Is there a reason you're using this formatting syntax over the f-strings we had before here? I personally find the f-strings a bit easier to understand

Reply:
It was to suppress this warning from pylint: 
```
W1202: Use lazy % formatting in logging functions (logging-format-interpolation
```
AFAIK lazy formatting is more efficient for complex formatting operations. Probably not a big deal this tool, but I chose to use it here since I actually find this format _more_ clear than the slightly weird syntax needed to define the padding in the f-strings! When editing the code I would get confused about where the padding was getting applied 🤷‍♀️